word-guess.rb

```diff
@@ -0,0 +1,97 @@
+
+require 'colorize'
+require 'Faker'
+class Game
+  attr_reader  :name
+ def initialize (name)
+   @name = name
+ end
+  def empty_array?(array)
+    array.each do |value|
+      return false if value != ""
+    end
+  end
+
+  def display_underlines(array_of_underlines)
+    print "Word: "
+    array_of_underlines.each do |underline|
+      print underline
+    end
+    puts
+  end
+
+  def input_incorrect?(user_guess, random_word)
+    special = "@#%!*1234567890-=+()"
+    if special.include?(user_guess)
+      return true
+    end
+  end
+
+  def find_place (random_word, user_guess, array_of_underlines)
+    random_word.each_with_index do |char, index|
+      if char == user_guess
+        array_of_underlines[index] = user_guess # put guessed letter in array of underlines
+        random_word[random_word.index(user_guess)] = ''
+      end
+    end
+  end
+
+  def play_game
```

Comment:
In your PR, you asked about how this code could be better organized. Here are a few suggestions:

1. Make playing a round its own method. The code should be independent enough from the logic of starting and stopping a game that it wouldn't be too difficult to rip out.
1. Use empty lines to separate different pieces of code, similar to how you would space paragraphs in an essay. I find that 3 to 5 lines is usually the right amount to group together.
1. Use a comment, on a line by itself, to clarify any code that's not very clear.